com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -144,7 +152,7 @@ public static void DrawMultipleFields(GUIContent label, SerializedProperty[] ppt
 
                 using (new EditorGUILayout.VerticalScope())
                 {
-                    EditorGUIUtility.labelWidth = 40;
+                    EditorGUIUtility.labelWidth = GetLongestLabelWidth(lbls) + 2; // Add some margin
```

Comment:
why not add a Space(2) instead?

Reply:
I didn't know about it :) Thanks

Reply:
done, added a new constant on CoreEditorStyles